parm/config/config.defaults.s2sw

```diff
@@ -17,21 +17,22 @@ use_cice_alb=".true."
 
 # config.fv3
 DELTIM=300
-layout_x_gfs=8
-layout_y_gfs=8
+layout_x_gfs=24
+layout_y_gfs=16
 WRITE_GROUP_GFS=1
-WRTTASK_PER_GROUP_GFS=24
+WRTTASK_PER_GROUP_GFS=86
```

Comment:
I want to make sure that these increases are only for S2S prototypes. 

Are these reasonable resources? Have these been vetted to make it to the `develop` branch?
These seem off by a lot. 
Before: 6x8x8 + 1x24 = 408
This PR: 6x24x16 + 1x86 = 2390

What is the change in throughput with this increase?

Reply:
This is surprising to request almost 6 times resources for the same resolution test. We seem not seeing this resource change in RT. May I ask the reason to increase both forecast and write tasks? Can someone show the computational performance? Also I'd curious how the number of 86 is decided for write tasks, should it be 96? This number may cause load balance issue on write tasks. I'd suggest to increase write groups, rather than the write tasks (e.g. write_groups=4, wrttask_per_group=24).

Reply:
@junwang-noaa are the RT's set up to run 35 day runs in under 8 hours?  

These resources will allow for S2SWA to run 35 days in about 7-7.5 hours.   This is not a perfect load-balancing, but it works.  @aerorahul my load balancing results are on hera so I cannot share them until hera is back online. 

It seems like there is a lot of opposition to these resources even though these are very similar to what was run is p8b: https://github.com/NOAA-EMC/global-workflow/blob/prototype/8b/parm/config/config.defaults.s2sw#L20-L21  I welcome others to run load-balancing tests to find a more optimal solution or to let me know what an acceptable default setting is.  I stopped before being finished doing a complete load-balancing as a set of resources was found to meet our immediate goals of P8 and more load balancing will be required after new features such as FMS mixed mode and ESMF threading are available.   It seems like a workable compromise will be to revert these two lines of code and put in a comment what resources will be used for P8c/P8 runs.   This will be committed shortly.

Reply:
@JessicaMeixner-NOAA To clarify, I am not saying using RT to run 35 days under 8 hours. I am saying we are using RT to monitor the resource change in the model updates.